starter-code/french-onion-soup-starter-code/index.html

```diff
@@ -1,43 +1,53 @@
-
-  We have been trying French onion soup in restaurants for years and my family
-and friends agree none can compare to my recipe for taste and simplicity of
-preparation.
-
-Ingredients
-1/2 cup unsalted butter
-2 tablespoons olive oil
-4 cups sliced onions
-4 (10.5 ounce) cans beef broth
-2 tablespoons dry sherry (optional)
-1 teaspoon dried thyme
-salt and pepper to taste
-4 slices French bread
-4 slices provolone cheese
-2 slices Swiss cheese, diced
-1/4 cup grated Parmesan cheese
 
-Directions
-Melt butter with olive oil in an 8 quart stock pot on medium heat. Add onions and
-continually stir until tender and translucent. Do not brown the onions.
-
-Add beef broth, sherry and thyme. Season with salt and pepper, and simmer for 30
-minutes.
+<body>
+  <header id="site-header" class="container">
+    <div class="container">
+      <h1>Onion Soup</h1>
+    </div>
+  </header>
+  <div class="section container">
+  <span class="larger-paragraph">We have been trying French onion soup in restaurants for years and my family and friends agree none can compare to my recipe
+  for taste and simplicity of preparation. </span>
+<img src="images/recipe-info.png" alt="Recipe info" class="img-center">
+  <div class="line container"></div>
 
-Heat the oven broiler.
+  <div class="section container">
+  <h2>Ingredients</h2> 
+  <li>1/2 cup unsalted butter</li> 
+  <li>2 tablespoons olive oil </li>
+  <li>4 cups sliced onions</li>
+  <li>4 (10.5 ounce) cans beef broth </li>
+  <li>2 tablespoons dry sherry (optional) </li>
+  <li>1 teaspoon dried thyme salt and pepper to taste 4 slices</li>
+  <li>French bread </li>
+  <li>4 slices provolone cheese </li>
+  <li>2 slices Swiss cheese, diced </li>
+  <li>1/4 cup grated Parmesan cheese</li>
+  <img src="images/cooking-info.png" alt="Cooking info" class="img-center">
+  </div>
+  <div class="line container"></div>
 
-Ladle soup into oven safe serving bowls and place one slice of bread on top of each
-(bread may be broken into pieces if you prefer). Layer each slice of bread with a slice of
-provolone, 1/2 slice diced Swiss and 1 tablespoon Parmesan cheese. Place bowls on
-cookie sheet and broil in the preheated oven until cheese bubbles and browns slightly.
+  <div class="section container">
+   <h2>Directions</h2> 
+   <ol><li>Melt butter
+  with olive oil in an 
+  8 quart stock pot on medium heat. 
+  Add onions and continually stir until tender and translucent. Do
+  not brown the onions.</li> <li>Add beef broth, sherry and thyme. Season with salt and pepper, and simmer for 30 minutes. </li><li>Heat the
+  oven broiler.</li></li> <li>Ladle soup into oven safe serving bowls and place one slice of bread on top of each (bread may be broken
+  into pieces if you prefer). Layer each slice of bread with a slice of provolone, 1/2 slice diced Swiss and 1 tablespoon
+  Parmesan cheese. Place bowls on cookie sheet and broil in the preheated oven until cheese bubbles and browns slightly.</li></ol>
+  </div>
```

Comment:
Here the code looks a bit messy and has extra closing tags that you don't need. Here's how it would look like fixed:
```
<div class="section container">
  <h2>Directions</h2>
  <ol>
    <li>Melt butter with olive oil in an 8 quart stock pot on medium heat. Add onions and continually stir until tender and translucent.
      Do not brown the onions.</li>
    <li>Add beef broth, sherry and thyme. Season with salt and pepper, and simmer for 30 minutes. </li>
    <li>Heat the oven broiler.</li>
    <li>Ladle soup into oven safe serving bowls and place one slice of bread on top of each (bread may be broken into pieces
      if you prefer). Layer each slice of bread with a slice of provolone, 1/2 slice diced Swiss and 1 tablespoon Parmesan
      cheese. Place bowls on cookie sheet and broil in the preheated oven until cheese bubbles and browns slightly.</li>
  </ol>
</div>